src/model/movie.js

```diff
@@ -58,6 +65,90 @@ class Films extends AbstractObserver {
   getFilms() {
     return this._films;
   }
+
+  static adaptCommentToClient(comment) {
+    const clientComment = { ...comment };
+
+    clientComment.text = comment.comment;
+    clientComment.date = new Date(comment.date);
+    delete clientComment.comment;
```

Comment:
```suggestion
    const clientComment = { ...comment };

    clientComment.text = comment.comment;
    clientComment.date = new Date(comment.date);
    delete clientComment.comment;
```
так засунь сразу все что тебе надо в clientComment, без всяких удалений и тд. Явно опиши все поля

Reply:
В других местах тоже. Просто собери тот объект что тебе надо